gen3/neural-networks/advanced-examples/image-matching/xfeat/README.md

```diff
@@ -0,0 +1,57 @@
+# XFeat: Accelerated Features for Lightweight Image Matching
+
+## Overview
+
+This example demonstrates how to use the XFeat model from Luxonis HubAI with the DepthAI platform. XFeat has compact descriptors (64D) and simple architecture components that facilitate deployment on embedded devices. Performance is comparable to known deep local features such as SuperPoint while being significantly faster and more lightweight.
+
+The model used in the example is available in our HubAI Model ZOO [here](https://hub.luxonis.com/ai/models/6c2790a1-bf68-4e89-a4b3-5c9ae68183b5?view=page).
+
+We offer two modes of operation: `mono` and `stereo`. In `mono` mode we use a single camera as an input and match the frames to the reference image. In `stereo` mode we use two cameras to match the frames to each other.
+In `mono` mode we visualize the matches between the frames and the reference frame which can be set by pressing `s` key in the visualizer. In `stereo` mode we visualize the matches between the frames from the left and right camera.
+
+## Demo
+    
+![XFeat: Accelerated Features for Lightweight Image Matching](https://drive.google.com/thumbnail?id=15P1N52ZhRpKkgJYD-acT4BQ2qPq-0PKU&sz=w1000)
+
+## Installation
+
+You need to prepare a Python 3.10 environment with [DepthAI](https://pypi.org/project/depthai/) and [DepthAI Nodes](https://pypi.org/project/depthai-nodes/) packages installed. You can do this by running:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Usage
+
+```
+python3 main.py --model <MODEL> --device <DEVICE> --fps_limit <FPS_LIMIT>
+```
+
+
+- `<MODEL>`: Model slug from Luxonis HubAI. Default: `luxonis/xfeat:mono-240x320`.
```

Comment:
Rename `Model slug` -> `HubAI Model Reference` (this is how we changed in it generic example)

Reply:
Fixed in [6a13bcf](https://github.com/luxonis/depthai-experiments/pull/562/commits/6a13bcf12caa9601e10c84b69b8ecb2e7a50b78e).